README.md

```diff
@@ -1,6 +1,7 @@
-# Serial interface to DYP-AO1 ultrasonic sensor
+# Serial interface to DYP-A01 ultrasonic sensor
 
 A simple serial interface to the inexpensive DYP-A01 ultrasonic ranger, available from Adafruit and AliExpress.
+
```

Comment:
The newline helps the package-manager to find the description of the package.
This is the line that is used by the package registry server.